### ISSUE TITLE
Could a potential timeline be added?

### ISSUE THREAD

**Issue by SokolskyNikita** [flattenthecurve/guide#89]
Please report the issue under the appropriate header below.

**Content to be added or changed:**
The website goes to a great length to explain the concept of "flattening the curve". However it does not provide an exact breakdown of how long this process would take, as the graphs presented lack an exact timescale.

Would it be possible to add such a timescale? E.g. start with the number of available hospital beds in the US and total hospitalization rates to calculate how many people could be infected with COVID-19 at the same time without overflowing the system. My own rough calculation is as following:

Total beds available: [924,000](https://www.boston.com/news/health/2020/03/14/coronavirus-math)

Estimate on beds that could be available for COVID-19 patients: 50% or ~500,000

Percentage of critical cases: [1%, according to South Korean case data](https://www.worldometers.info/coronavirus/country/south-korea/). South Korea did the most testing so they've captured a lot more asymptomatic patients than others.

Percentage of US population that needs to have immunity for "herd immunity" to kick in: ~70% or 228 million

Median hospital stay duration: 3 weeks

From this we can derive that:

Number of Americans that can be infected at the same time: 500,000 * (1 / 0.01) = 50 million

Number of weeks required to get to "herd immunity" stage: (228 / 50) * 3 = 13.68 or 3.5 months

If my math makes sense, I can provide a pull request as well that will add the numbers to your main chart.

**Comment by drlbln** [comment]
Good point, as long as we can provide proper references and can we please pay attention to prepare those graphs as reusable templates so we can easily adapt them to other setups?

**Comment by SokolskyNikita** [comment]
@drlbln looks like someone [already did the math](https://medium.com/@joschabach/flattening-the-curve-is-a-deadly-delusion-eea324fe9727). Can I include those graphs in the repository, presuming the Medium author allows it? The math looks solid to me.

**Comment by rousik** [comment]
My understanding is that the site is aiming to provide practical information about what-to-do and rather linking to external pages/sources for information that evolves and changes. We did link to some of the "official" models outlining the course of the outbreak (one being https://covid19.healthdata.org/projections).

My understanding is that lockdowns/stay-at-home and long term strategy is one thing that is very uncertain, rapidly evolving and volatile.

I don't think we really have capacity to keep this piece information fresh and up to date and decide which of the many different projections we are going to go with. I believe that there are downsides to having outdated content on the site.

So I think we should probably opt for not including this piece of information directly on the website. We can absolutely link to those analyses via our Resources.

Note that  the original unlabeled flatten-the-curve graph is no longer present on the website so this may be another reason for not including this.

Adding @jmcmurry for visibility.

**Comment by SokolskyNikita** [comment]
The website still includes a misleading unlabeled flatten the curve graph here: https://www.flattenthecurve.com/covid-19/. It should be amended to showcase exactly how long it would take for everyone to be slowly infected without overwhelming the hospitals. 

There is now data from NYC (21% have antibodies) which makes this fairly easy to calculate.

**Comment by rousik** [comment]
Thanks for pointing this out. I do believe our COVID-19 section needs a bit of overhaul and updating.

Back on the topic of herd immunity. I should disclose that I have no epidemiology/public health background. My expertise is dev-ops incident response so that's what influences my thinking and analysis.

I'm not sure if reaching herd immunity as fast as possible w/o overwhelming medical system is the goal we should be aiming for. It is better than overwhelming our medical system but will still result in massive casualties before we get there. Assuming that IFR is in the 0.3-1% range ([WHO report](https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200219-sitrep-30-covid-19.pdf?sfvrsn=3346b04f_2) and that we need to reach 70% of population, this translates to 0.6M-2.3M lives lost in US and 15-53M deaths globally.

I hope we can do better than this by controlling the outbreaks, contact tracing/isolation and basically buying us as much time as needed before we can come up with effective interventions in forms of treatment and/or vaccination.

The site says *the most important thing to do is flatten the curve of the epidemic so that our health systems can cope and to give time for the scientists to research vaccines and treatments*. The picture implies that we want to ride-it-out at the peak healthcare system utilization but I'm not sure this is sufficient and that we should aim to "give more time for the scientists to research vaccines and treatments".

I hope this is seen as constructive conversation and I should point out that this is my personal opinion. I hope we will find agreement and right messaging around what is needed to manage this public crisis effectively.

**Comment by SokolskyNikita** [comment]
@rousik The problem with this argument is that it completely goes against what "flatten the curve" was supposed to be. The strategy you're describing is "crush the curve", where we not only get below what the medical system can handle, but rather focus on saving as many lives as possible, even if we have hundreds of thousands of spare beds available (as is the case in the US right now). If this is the case, it would probably be best to rename the project to "crush the curve" or something equivalent, as otherwise it is misleading its readers.

**Comment by rousik** [comment]
Yeah. I agree. These were my thoughts and I don't know how much in line with opinions of others and with the overall intent of this site. I'm just voicing my concerns with a strategy of barely maintaining the curve just below the medical system capacity.

I will try to loop in more people to this conversation so that we can find some consensus and see if we can clarify the messaging.

Thank you very much for your input so far!

**Comment by SokolskyNikita** [comment]
@rousik I've added a new Issue to discuss this: https://github.com/flattenthecurve/guide/issues/372

**Comment by rousik** [comment]
Perfect, let's move the discussion there.
